simtools/data_model/validate_data.py

```diff
@@ -502,11 +507,16 @@ def _get_reference_data_column(self, column_name, status_test=False):
         ----------
         column_name: str
             Column name.
+        status_test: bool
+            Test if reference column exists.
 
```

Comment:
There are many more parameters not listed here. It would be good to have them here, even if the function is internal

Reply:
Help me: the function call is `_get_reference_data_column(self, column_name, status_test=False)` and both variables are listed. What is missing?

Reply:
Ok, I think I looked somewhere else, sorry.